worker/lib/workers/qemu.ts

```diff
@@ -294,24 +326,42 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 	public async network(configuration: {
 		wired?: { nat: boolean };
 	}): Promise<void> {
-    const bridgeName: string = this.qemuOptions.network.bridgeName;
-    const bridgeAddress: string = this.qemuOptions.network.bridgeAddress;
+	
+	// generate random bridge name 
+	const id = `${Math.random()
+		.toString(36)
+		.substring(2, 10)}`;
+	this.bridgeName =  `br${id}`;
+
+	// generate bridge subnet
+	// e.g '192.168.100.X'
+	let ip = await this.getBridgeIpAddress();
+
+	// Assign ip address to bridge
+	// e.g '192.168.100.1'
+	this.bridgeAddress = `${ip}.1`;
+
+	// generate dhcpRange for dnsmasq
+	// e.g '192.168.100.2,192.168.100.254'
+	this.dhcpRange = `${ip}.2,${ip}.254`;
```

Comment:
We should use a node module to calculate this from the provided subnet.

`ipcalc` is a nice utility for Linux but there must be a node equivalent so we aren't hardcoding assumptions about networks.

Reply:
we've decided to keep it simple for now and assume the range 2-254 is available in the last octet